.github/workflows/hubble-cli-integration-test.yaml

```diff
@@ -206,6 +211,8 @@ jobs:
     if: ${{ always() }}
     needs: integration-test
     uses: ./.github/workflows/common-post-jobs.yaml
+    permissions:
+      {}
```

Comment:
Interesting, I guess this implies that the reusable action `./.github/workflows/common-post-jobs.yaml` adds the permissions from that workflow on top of the permissions requested here?

Reply:
^^ Is the above true or do we need to test it?